pkgs/default.nix

```diff
@@ -12,6 +12,7 @@
   nixops19_09 = pkgs.callPackage ./nixops { };
   netns-exec = pkgs.callPackage ./netns-exec { };
   lightning-loop = pkgs.callPackage ./lightning-loop { };
+  btcpayserver = pkgs.callPackage ./btcpayserver { };
```

Comment:
`nix build -f . btcpayserver` didn't work for me, I had to do:

```diff
diff --git a/pkgs/default.nix b/pkgs/default.nix
index f87f774..0dbb183 100644
--- a/pkgs/default.nix
+++ b/pkgs/default.nix
@@ -1,5 +1,5 @@
 { pkgs ? import <nixpkgs> {} }:
-{
+rec {
   lightning-charge = pkgs.callPackage ./lightning-charge { };
   nanopos = pkgs.callPackage ./nanopos { };
   spark-wallet = pkgs.callPackage ./spark-wallet { };
@@ -12,7 +12,8 @@
   nixops19_09 = pkgs.callPackage ./nixops { };
   netns-exec = pkgs.callPackage ./netns-exec { };
   lightning-loop = pkgs.callPackage ./lightning-loop { };
-  btcpayserver = pkgs.callPackage ./btcpayserver { };
+  nbxplorer = pkgs.callPackage ./nbxplorer { };
+  btcpayserver = pkgs.callPackage ./btcpayserver { inherit nbxplorer; };
 
   pinned = import ./pinned.nix;
 }

```

Reply:
I solved this problem by adding 

```diff
diff --git a/pkgs/default.nix b/pkgs/default.nix
index f87f774a47a..6209b00dba9 100644
--- a/pkgs/default.nix
+++ b/pkgs/default.nix
@@ -12,6 +12,7 @@
   nixops19_09 = pkgs.callPackage ./nixops { };
   netns-exec = pkgs.callPackage ./netns-exec { };
   lightning-loop = pkgs.callPackage ./lightning-loop { };
+  nbxplorer = pkgs.callPackage ./nbxplorer { };
   btcpayserver = pkgs.callPackage ./btcpayserver { };
```
and removing
```diff
diff --git a/pkgs/btcpayserver/default.nix b/pkgs/btcpayserver/default.nix
index 0acb62dd447..dcf5a3ec42d 100644
--- a/pkgs/btcpayserver/default.nix
+++ b/pkgs/btcpayserver/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, fetchurl, nbxplorer, dotnetPackages, dotnetCorePackages, lib }:
+{ stdenv, fetchFromGitHub, fetchurl, dotnetPackages, dotnetCorePackages, lib }:
```
and then called both packages from the module. I think this is the better solution as someone might want to use NBXplorer without BTCPayServer.

Reply:
Applied @nixbitcoin's solution